dbrequests/mysql/tests/test_mysql_connection.py

```diff
@@ -206,3 +208,25 @@ def test_escape_sequences(self, db):
 
         assert res.name[0] == '\\'
         assert res.owner[0] == 'a'
+
+    def test_update_json_and_decimal(self, db):
+        """Insert None/NULL values for json and decimal types: #30"""
+        def is_none(x):
+            """Check that a value is None. == is violating some pep."""
+            if x:
+                return np.isnan(x)
+            return True
```

Comment:
`x is None` should be used instead of `x == None`. Though  it is not working with np.nan values(`np.nan is None` -> False) so your solution is good.
Is it ok to define the function inside the function or should it be moved into a utils file?

Reply:
I resolved it, and now it is a bit less of a surprise. Nested function definitions are legit in python, however not necessary in the new version I committed.